src/main/asciidoc/product/book_info.adoc

```diff
@@ -0,0 +1,19 @@
+
+= Functionality of Narayana supported within Red Hat JBoss EAP
+
+:leveloffset: 1
+include::author_group.adoc[]
+:leveloffset: 0
+
+
+xref:fallback_content/legal_notice.adoc[Legal Notice]
+
+=== Abstract
+
+The Transactions Overview Guide contains information on how to use {PRODUCT};
```

Comment:
Could you please change all occurrences of {PRODUCT};} with {productName}? Thanks
```suggestion
The Transactions Overview Guide contains information on how to use {productName}
```